compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -1209,6 +1209,10 @@ void OMR::X86::CodeGenerator::saveBetterSpillPlacements(TR::Instruction * branch
 
 void OMR::X86::CodeGenerator::removeBetterSpillPlacementCandidate(TR::RealRegister * realReg)
    {
+   // this mechanism only supports GPR's
```

Comment:
Can you drop a sentence in here explaining why so it is clear to others coming across this code?  (presumably interference with the register mask)

Reply:
See [force-push](https://github.com/eclipse/omr/compare/86ebbed2aa05171a1403da6f76b5d246953adf09..755c5e345eb05a3612e4b19e48440b1e7c61ea0f)